unison-src/transcripts-round-trip/main.output.md

```diff
@@ -771,7 +771,7 @@ a |> f = f a
 This diff should be empty if the two namespaces are equivalent. If it's nonempty, the diff will show us the hashes that differ.
 
 ```ucm
-.> diff.namespace a1 a2
+scratch/main> diff.namespace /a1:. /a2:.
```

Comment:
Would like to be able to `diff.namespace /a1 /a2`; would that already work?

Reply:
Unfortunately not, that'll require a few edits to the diff.namespace command and probably some input parsers. We can do that as a fast-follow-up